frontend/app/src/App.tsx

```diff
@@ -614,25 +616,17 @@ export class App extends PureComponent<Props, State> {
     }
 
     if (favicon) {
+      this.hostCommunicationMgr.sendMessageToHost({
+        type: "SET_PAGE_FAVICON",
+        favicon: getFaviconUrl(favicon, this.endpoints),
+      })
+
       if (this.state.hostConfig.disableSetPageMetadata) {
         logError(
           "Setting the page favicon is disabled in line with the platform security policy."
         )
-        handleFavicon(
-          favicon,
-          this.hostCommunicationMgr.sendMessageToHost,
-          this.endpoints,
-          // do not overwrite the favIcon but still send the message for debugging purposes
-          false
-        )
       } else {
-        handleFavicon(
-          favicon,
-          this.hostCommunicationMgr.sendMessageToHost,
-          this.endpoints,
-          // overwrite the favIcon
-          true
-        )
+        overwriteFavicon(favicon)
```

Comment:
I think we need to use `getFaviconUrl(favicon, this.endpoints)` here as well, or?

Reply:
Yep thanks. Fixed.